ethereum/.env.example

```diff
@@ -6,3 +6,9 @@ MNEMONIC="candy taste cake sugar pudding cream waffles rich smooth crumble sweet
 
 # INFURA_PROJECT_ID is the project's Infura ID
 INFURA_PROJECT_ID=e8b4790b8e4049cca3c04f738cfa25f2
+
+MNEMONIC
+INFURA_PROJECT_ID
+BASE_PATH
+ETH_ENDPOINT
+SUB_ENDPOINT
```

Comment:
There are inconsistency between .env and .env.example:
.example contains `SUB_ENDPOINT` while .env contains `SUB_ENDPOINT_WS`, same for ETH_ENDPOINT.
Some variables from .example are not seen in .env

Reply:
.env.example is an old file that should be deleted. I will delete it.